ambari-metrics/ambari-metrics-timelineservice/pom.xml

```diff
@@ -738,6 +737,25 @@
       <version>3.2</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-minicluster</artifactId>
+      <version>${hadoop.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-client</artifactId>
+      <version>${hadoop.version}</version>
+    </dependency>
+
+    <dependency>
```

Comment:
The commons-beanutils-1.9.3.jar located in ambari-metrics-assembly/target/embedded/hadoop-3.1.1/share/hadoop/common/lib/
Maybe we could use that instead of introducing a new managed dependency here.

Reply:
I see you are right, I will use that instead.